deeplearning4j/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/preprocessing/text/KerasTokenizer.java

```diff
@@ -114,7 +114,7 @@ public static KerasTokenizer fromJson(String jsonFileName) throws IOException, I
             throw new InvalidKerasConfigurationException("No configuration found for Keras tokenizer");
 
 
-        int numWords = (int) tokenizerConfig.get("num_words");
+        Integer numWords = (Integer) tokenizerConfig.get("num_words");
```

Comment:
In addition to fixing the NPE, this avoids an unnecessary unbox-then-immediately-rebox. (Same pattern is in use for `documentCount` and could also be changed.)